monitoring/docker/provisioning/dashboards/dashboards.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: 1
+
+providers:
+- name: 'a unique provider name'
```

Comment:
can we add Provider name?

Reply:
@mankenavenkatesh Please suggest a name for the same.